packages/vscode-extension/media/itp/itp.md

```diff
@@ -59,5 +61,3 @@ If you don't have a personal Microsoft account, or a work or school account. You
 ![Instant sandbox](./turn-on-sideload.png)
 
 - Select `Save`.
-
->Note: It can take up to 24 hours for the sideloading option to be active.
```

Comment:
Does this mean user doesn't need to wait for 24h before being able to use sideloading?

Reply:
Let's check with PM.